integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -98,6 +110,48 @@ private static void recoverGravitinoServerConfig() throws IOException {
     Files.move(tmpPath, configPath);
   }
 
+  protected static void downLoadMySQLDriver(String relativeDeployLibsPath) throws IOException {
+    if (!ITUtils.EMBEDDED_TEST_MODE.equals(testMode)) {
+      String gravitinoHome = System.getenv("GRAVITINO_HOME");
+      java.nio.file.Path tmpPath = Paths.get(gravitinoHome, relativeDeployLibsPath);
+      JdbcDriverDownloader.downloadJdbcDriver(DOWNLOAD_JDBC_DRIVER_URL, tmpPath.toString());
+    }
+  }
+
+  private static void setMySQLBackend() {
+    String mysqlUrl = MYSQL_CONTAINER.getJdbcUrl();
+    customConfigs.put(Configs.ENTITY_STORE.getKey(), "relational");
+    customConfigs.put(Configs.ENTITY_RELATIONAL_STORE.getKey(), "JDBCBackend");
+    customConfigs.put(Configs.ENTITY_RELATIONAL_JDBC_BACKEND_URL.getKey(), mysqlUrl);
+    customConfigs.put(
+        Configs.ENTITY_RELATIONAL_JDBC_BACKEND_DRIVER.getKey(), "com.mysql.cj.jdbc.Driver");
+    customConfigs.put(Configs.ENTITY_RELATIONAL_JDBC_BACKEND_USER.getKey(), "root");
+    customConfigs.put(Configs.ENTITY_RELATIONAL_JDBC_BACKEND_PASSWORD.getKey(), "root");
+
+    LOG.info("MySQL URL: {}", mysqlUrl);
+    // Connect to the mysql docker and create a databases
+    try (Connection connection =
+        DriverManager.getConnection(
+            StringUtils.substring(mysqlUrl, 0, mysqlUrl.lastIndexOf("/")), "root", "root")) {
+
+      final Statement statement = connection.createStatement();
```

Comment:
Maybe we can put this line in the try-with-resources statement to close statement automatically.

Reply:
OK